src/bodywork/cli/cli.py

```diff
@@ -371,25 +350,23 @@ def deployment(args: Namespace) -> None:
 
     :param args: Arguments passed to the deploy command from the CLI.
     """
-    command = args.command
-    namespace = args.namespace
     name = args.name
+    command = args.command
     retries = args.retries
     git_repo_url = args.git_repo_url
     git_repo_branch = args.git_repo_branch
     run_workflow_controller_locally = args.local_workflow_controller
-    if (
-        command == "create" or command == "logs" or command == "delete_job"
-    ) and name == "":
-        print("please specify --name for the deployment")
-        sys.exit(1)
+
     if command == "create" and git_repo_url == "":
         print("please specify Git repo URL for the deployment you want to create")
         sys.exit(1)
+    if command != "create" and name == "":
+        print("please specify --name for the job")
+        sys.exit(1)
     if command == "create":
         if run_workflow_controller_locally:
             pass_through_args = Namespace(
-                namespace=namespace,
+                namespace=BODYWORK_DEPLOYMENT_JOBS_NAMESPACE,
```

Comment:
I don't think we need to pass this through anymore as it's not used in `workflow`?